docs/release.md

```diff
@@ -39,79 +21,61 @@ These artifacts are provided in their respective channels and their natural way
-| 1                   | 1.24               | 1 (January 03)   | 0.55                    | 2 (January 10)        |                                             |
-| 15                  | 1.24               | 15 (April 12)    | 0.55                    | 16 (April 19)         |                                             |
-| 17                  | 1.25               | 1 (May 23)       | 0.56                    | 2 (May 30)            | KubeCon + CloudNativeCon EU likely to occur |
-| 32                  | 1.25               | 15 (August 22)   | 0.56                    | 16 (August 29) - 19 (September 26)        |                                             |
-| 34                  | 1.26               | 1 (September 5)  | 0.57                    | 2 (September 12)      | KubeCon + CloudNativeCon NA likely to occur |
-| 49                  | 1.26               | 14 (December 05) | 0.57                    | 15 (December 12) - 18 (January 9)    |                                             |
-
-Initial release candidates are cut **every 4 weeks.**
-With a 15 week cadence, there will be at least 3 release candidates:
-- v*-rc.0
-- v*-rc.1
-- v*-rc.2
-
-The stable branch will be created when the -rc.2 is tagged, 12 weeks into
-the release cycle.  Once the branch is created, only backports will be
+
+# Minor releases
+
+KubeVirt aligns to the [Kubernetes release cycle](https://www.kubernetes.dev/resources/release/).
+But because KubeVirt builds upon Kubernetes, it will trail Kubernetes' release schedule by roughly 2 months, in order to have time to consume and stabilize on top of new releases.
+
+Thus new release branches are cut 3 times a year, or about every 15 weeks.
+
+## Guiding principles
+
+The release schedule is build around a few guiding principles:
+
+- Alpha and Beta tags are applied on main
+- Release branches must only be cut after a Kubernetes minor releases
+- RCs must only be tagged on the release branch
+
+## CI Provider
+
+A pre-condition for the GA of a KubeVirt release is the presence of a KubeVirt CI provider for the correspoding Kubernetes release.
+Around the beta release of the corresponding Kubernetes release, the KubeVirt team will start to work on a KubeVirt CI provider.
+The assumption is, that the KubeVirt CI provider will be available by KubeVirt's enhcanement freeze.
+
+## Release schedule schema  
+
+This can then be translated into the following KubeVirt release schedule schema:
+
+| Week Number in Kubernetes Release Cycle | KubeVirt Milestone | Note                            |
+|:---------------------------------------:|--------------------|---------------------------------|
+| 14 - 8                                  | Alpha              | KubeVirt alpha.0 on main        |
+| 14 - 4                                  | Beta               | KubeVirt beta.0 on main         |
```

Comment:
I know this already merged. And i think it's definitely a good revision.

One think I might want to iterate on in the future is the cadence for alpha releases. I'd us to keep making some sort of release out every month even it is super early in the dev cycle.